install_config/registry/registry_known_issues.adoc

```diff
@@ -67,9 +67,11 @@ the `no_wdelay` options listed. The `no_wdelay` option prevents the server from
 delaying writes, which greatly improves read-after-write consistency, a
 requirement of the registry.
 
-[NOTE]
+[IMPORTANT]
 ====
-include::install_config/topics/NFS_note.adoc[]
+Testing has shown issues with using the NFS server on RHEL as storage backend for the container registry. This includes the embedded registry and Quay, Cassandra for metrics storage, and ElasticSearch for logging storage. Therefore, using NFS on RHEL as storage for these framework services is not recommended.
```

Comment:
We can remove the references to Cassandra, and Elastic Search here.

The section this is in, implies we will not be discussing those components.

Reply:
I removed these in the new PR.